features/topologies/service/api/src/main/java/org/opennms/netmgt/topologies/service/api/OnmsTopologyVertex.java

```diff
@@ -28,14 +28,13 @@
 
 package org.opennms.netmgt.topologies.service.api;
 
-public class OnmsTopologyVertex extends OnmsTopologyAbstractRef implements OnmsTopologyRef {
+import java.util.Objects;
 
+public class OnmsTopologyVertex extends OnmsTopologyAbstractRef implements OnmsTopologyRef {
 
-    public static OnmsTopologyVertex create(String id,String label,String address,String iconKey) throws OnmsTopologyException {
-        if (id == null) {
-            throw new OnmsTopologyException("id is null, cannot create vertex");
-        }
-        return new OnmsTopologyVertex(id,label,address,iconKey);
+    public static OnmsTopologyVertex create(String id, String label, String address, String iconKey) {
+        return new OnmsTopologyVertex(Objects.requireNonNull(id, "id is null, cannot create vertex"), label, address,
```

Comment:
Can we put the `Objects.requireNonNull(...)` at the beginning of the method? This makes it easier to read.

Reply:
Will do.